01-front-end-basics/jquery-dom-manipulation/24/README.md

```diff
@@ -0,0 +1,252 @@
+#<img src="https://cloud.githubusercontent.com/assets/7833470/10423298/ea833a68-7079-11e5-84f8-0a925ab96893.png" width="60"> Intro DOM Manipulation & jQuery
+
+| Objectives |
+| :--- |
+| Explain the benefits of using jQuery for DOM manipulation |
+| Identify the main differences between jQuery and Vanilla JS |
+| Select DOM elements using CSS selectors and jQuery |
+| Use jQuery to dynamically change the DOM |
+
+## What is the DOM?
+
+DOM stands for **Document Object Model**. It's a fancy way of describing what's going on in your browser when you visit a website. You may think a webpage is just a chunk of HTML, CSS, and JavaScript code. But, there's an important step between a webpage being pure data (text) and being a rendered page users can explore. Everything you interact with in the browser can be represented as a JavaScript object. For example, this paragraph is an object, and it's full of information about being an HTML `<p>` tag.
+
+### DOM Analogy: The Human Body
+
+* HTML = Skeleton
+* Javascript = Muscles, Brain, and Organs
+* CSS = Skin and Clothing
+
+![muscles](https://cloud.githubusercontent.com/assets/7833470/10700126/8e2997da-7970-11e5-96ba-1e5eaf793394.jpg)
+
+## Intro to jQuery
+
+jQuery is a **powerful, easy-to-use JavaScript library**. It was invented in 2005 and has made our lives as developers better ever since. The slogan for jQuery is "write less, do more". jQuery is a great toolbox that provides convenient functions we can use as opposed to writing the functionality we need from scratch.
+
+*We use jQuery because it's:*
+
+* **Convenient:** solves problems developers commonly face.
+* **Less Buggy:** ensures JavaScript DOM manipulation works the same, cross-browser.
+* **Modern:** brings JavaScript DOM manipulation into the 21st century.
+* **Popular:** 26.95% of all sites use jQuery!
+
+### jQuery vs. Vanilla JS
+
+It's important to note that any DOM manipulation done with jQuery can also be done with pure (vanilla) JavaScript. Some argue <a href="http://youmightnotneedjquery.com" target="_blank">you might not need jQuery</a>. It can be overkill, depending on what you're doing.
+
+Take a look at this <a href="http://bl.ocks.org/joyrexus/5322252" target="blank">Comparison of jQuery and Vanilla JS</a>. Here are some of the basic differences:
+
+**Selecting Elements**
+
+```js
+// jquery
+var $divs = $('div');
```

Comment:
Might be worth mentioning why this var starts with a peso, and why the other doesn't.